code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -110,21 +111,43 @@
 	for(var/mob/living/carbon/human/H in view(7, src))
 		if(H in breach_affected)
 			continue
+		breach_affected += H
 		if(HAS_TRAIT(H, TRAIT_COMBATFEAR_IMMUNE))
+			H.say(pick(list("Pff. It’s nothing.","I've got this.","Piece of cake.","Just tickles is all.","How boring.","Doesn’t even phase me.")))
 			continue
-		breach_affected += H
 		var/sanity_result = round(fear_level - get_user_level(H))
-		var/sanity_damage = -(max(((H.maxSanity * 0.26) * (sanity_result)), 0))
-		H.adjustSanityLoss(sanity_damage)
-		if(H.sanity_lost)
-			continue
+		var/sanity_damage
 		switch(sanity_result)
+			if(-INFINITY to 0)
+				continue
 			if(1)
+				sanity_damage = -(H.maxSanity*0.1)
+				H.apply_status_effect(/datum/status_effect/panicked_lvl_1)
+				if(H.sanity_lost)
+					continue
 				to_chat(H, "<span class='warning'>Here it comes!")
 			if(2)
+				sanity_damage = -(H.maxSanity*0.3)
+				H.apply_status_effect(/datum/status_effect/panicked_lvl_2)
+				if(H.sanity_lost)
+					continue
 				to_chat(H, "<span class='danger'>Not that thing...")
-			if(3 to INFINITY)
+			if(3)
+				sanity_damage = -(H.maxSanity*0.6)
+				H.apply_status_effect(/datum/status_effect/panicked_lvl_3)
+				if(H.sanity_lost)
+					continue
 				to_chat(H, "<span class='userdanger'>I'm not ready for this!")
+			else
+				if(panic_bypass) // Should certain things be able to drive people insane instantly by themselves? Most definitely. Should most things? ehh... It's not exactly fun to turn a corner and lose control because of it. Now if you get hit afterwards or see multiple? Shit out of luck.
```

Comment:
No. Just generalize the panic effect depending on fear_level.

Reply:
...it is? This is literally the make 99% panic thing that was talked about so Clerks and Low-Level agents don't instant panic to a single source from max sanity UNLESS it's one of a few specific sources?

Reply:
This is only even checked if a person is 4+ levels lower than the fear source.